spread.yaml

```diff
@@ -111,13 +111,13 @@ backends:
             - debian-sid-64:
                   workers: 6
 
-            - fedora-30-64:
-                  workers: 6
-                  manual: true
             - fedora-31-64:
                   workers: 6
```

Comment:
Aren't we dropping fedora 31 in the tests below and only testing 32 & 33 now?

Reply:
So apparently fedora-31 is not EOL just yet (will be soon), but I'm confused about why are we removing it from systems lines in the tests then.